bioblend/galaxy/users/__init__.py

```diff
@@ -186,3 +186,23 @@ def get_user_apikey(self, user_id):
         """
         url = self._make_url(user_id) + '/api_key/inputs'
         return self._get(url=url)['inputs'][0]['value']
+
+    def update_user(self, user_id, **kwds):
+        """
+        Update user information. Some of the attributes that can be
+        modified are documented below.
+
+        :type user_id: str
+        :param user_id: encoded user ID
+
+        :type username: str
+        :param username: Replace user name with the given string
+
+        :type email: str
+        :param email: Replace user email with the given string
+
+        :rtype: dict
+        :return: details of the updated user
+        """
+        url = self._make_url(user_id) + '/information/inputs'
```

Comment:
I see there is also a `PUT /api/users/{id}` endpoint (`update()` method in `lib/galaxy/webapps/galaxy/api/users.py`). I suppose this allows to change more attributes?

Reply:
I just tried with `PUT /api/users/{id}`, but no change is done in the db... Maybe the payload needs some special format?

Reply:
It seems only updating the username is supported in https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/managers/users.py#L586-L590

I think we should move the logic of user update from `UserAPIController.set_information()` to `UserDeserializer`, but for now this looks fine.